src/modules/bids-asks/actions/load-account-orders.js

```diff
@@ -1,18 +1,28 @@
+import async from 'async'
 import { augur } from 'services/augurjs'
 import logError from 'utils/log-error'
 import { addOpenOrderTransactions } from 'modules/transactions/actions/add-transactions'
-import { loadMarketsInfoOnly } from 'modules/markets/actions/load-markets-info'
+import { loadMarketsInfo } from 'modules/markets/actions/load-markets-info'
+import { updateOrderBook } from 'modules/bids-asks/actions/update-order-book'
 
 export const loadAccountOrders = (options, callback = logError) => (dispatch, getState) => {
   const { universe, loginAccount } = getState()
-  augur.trading.getOrders({ ...options, account: loginAccount.address, universe: universe.id }, (err, orders) => {
+  augur.trading.getOrders({ ...options, creator: loginAccount.address, universe: universe.id }, (err, orders) => {
     if (err) return callback(err)
     if (orders == null || Object.keys(orders).length === 0) return callback(null)
     const marketIDs = Object.keys(orders)
     // TODO: consolidate all the getting of maket infos for load account history
-    dispatch(loadMarketsInfoOnly(marketIDs.slice(), () => {
+    dispatch(loadMarketsInfo(marketIDs.slice(), () => {
       dispatch(addOpenOrderTransactions(orders))
-      // TODO update user's open orders
+      async.forEachOfSeries(marketIDs, (marketID, id, next) => {
```

Comment:
No need that I see for the async lib here, refactor this to just use the `forEach` method.  

While I’m not opposed to using external libs when a real utility is needed, when a base javascript comparable suffices, the preference is towards that in the CB.